src/modules/render_functions.js

```diff
@@ -0,0 +1,16 @@
+import { arrTasks } from './task_DatabaseMS.js';
+import { populateTaskList } from './task_UI_Manager.js';
+import { addIconEvent, addInputDescEvent } from './event_handlers.js';
+
+const renderUI = () => {
+  populateTaskList();
+  if (arrTasks.length === 0) {
+    for (let i = 0; i < arrTasks.length; i += 1) {
+      addIconEvent(arrTasks[i].index);
+      addInputDescEvent(arrTasks[i].index);
+      // addCheckBoxEvent(arrTasks[i].index);
```

Comment:
[ ] Kindly remove commented code from your project, This will make sure your project code is clean. You can leave comments about your code but not commented code It's not a good practice.